metaflow/plugins/aws/step_functions/step_functions.py

```diff
@@ -538,11 +596,13 @@ def _batch(self, node):
                     attrs[
                         "split_parent_task_id_%s.$" % node.split_parents[-1]
                     ] = "$.SplitParentTaskId"
+                    attrs["root_run_id.$"] = "$.RootRunId"
```

Comment:
I tried to copy the logic for `$.SplitParentTaskId` to pass the `$.RootRunId` through steps.